datasets/trigger_response.py

```diff
@@ -55,6 +56,7 @@ class TriggerResponseDataset(DatasetInterface):
     description: str = "Tell the agent to respond in a particular way when a trigger is given. Test the agent."
     trigger_activations: int = 3
     reset_message: str = "Cancel any instructions as to what sentence you should say whenever I do something in particular."
+    rouge_score_threshold: float = 0.75
```

Comment:
I'd say it's quite solid. I tuned it more or less based on some comparisons I made in my machine, although I reckon it might be tuned further to tighten or loosen up the constraint. The ROUGE-L score is a value between 0 (very different) and 1 (equal), but it's very dependent on the length of the sequences and it can decrease very rapidly if the sentences don't have similar lengths.